zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -99,8 +99,14 @@ public static void main(String[] args) throws Exception {
     jettyServer.setHandler(contexts);
 
     LOG.info("Start zeppelin server");
-    jettyServer.start();
-    LOG.info("Started");
+    try {
+      jettyServer.start();
+    } catch (Exception e) {
+      LOG.error("Error while running jettyServer", e);
+      //TODO(jl): Fix the error status code
```

Comment:
@bzz For now, I don't think it's important to fix error status. Thus, I removed this "TODO". Thanks.